src/test_tenax.py

```diff
@@ -21,14 +22,19 @@
 from pyTENAX.pyTENAX import *
 import time 
 import sys
+import matplotlib.pyplot as plt
+from scipy.stats import chi2
+
 
 #import smev 
 from pyTENAX.smev_class import *
 
 S = TENAX(
-        return_period = [2.33,5,10,20,30,50,100],
+        return_period = [1.1,1.2,1.5,2,5,10,20,50,100, 200],  #for some reason it doesnt like calculating RP =<1
```

Comment:
there cannot be lower than 1 year RP as the probabilities are calculated as "1 - 1 / np.array(target_return_periods) " , also from engineering perspective such RP are not useful cause we never design things that can stand only low RP.